db/migrate/20210806041206_users.rb

```diff
@@ -7,7 +7,6 @@ def change
       t.integer :age
       t.string :gender
 
-      t.timestamps
```

Comment:
@SampabloC  You shouldn't update previous migrations, why are you removing this line?